protocols/index.md

```diff
@@ -80,6 +81,9 @@ A highly secure, platform-independent enterprise filesharing solution
 ### [Files.com](files.com.md)
 Files.com is Smart Cloud Storage for modern teams
 
+### [SFTP To Go](sftptogo.md)
```

Comment:
In general we will not want to have a single provider in the top level protocol list. Exceptions apply.